spec/factories/sales_log.rb

```diff
@@ -80,7 +80,6 @@
       noint { 2 }
       privacynotice { 1 }
       age1_known { 0 }
-      staircase { 1 }
```

Comment:
Completed log has `ownershipsch: 2` which is discounted ownership. It cannot be staircase anyway cause only shared ownership can be staircased, but setting staircase to 1 here, clears some of the other fields and this log ends up being not completed (at least in 2025)